packages/bsky/src/services/label/index.ts

```diff
@@ -6,6 +6,10 @@ import { sql } from 'kysely'
 
 export type Labels = Record<string, Label[]>
 
+// @TODO forward these labels to client
+// these are labels are currently only used server side & not sent to client since it does not handle them correctly
+const SERVER_SIDE_LABELS = ['!no-promote']
```

Comment:
Also may be worth a little note that `SERVER_SIDE_LABELS` cannot be empty (otherwise query below would fail).